src/rpc/rawtransaction.cpp

```diff
@@ -237,6 +237,9 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
     }
     entry.push_back(Pair("vout", vout));
 
+    UniValue vjoinsplit = TxJoinSplitToJSON(tx);
+    entry.push_back(Pair("vjoinsplit", vjoinsplit));
+
```

Comment:
I understand the change and it looks fine to me. However I do not see any extra unit test/qa tests covering the change you made. I saw cronic opened issue 173 to ask for extra tests (it is marked as enhancement).
@MarcoOl94, @alsala  should we add the unit tests or qa tests covering this change?